### ISSUE TITLE
`node` deprecation warning: `In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead`

### ISSUE THREAD

**Issue by DilumAluthge** [julia-actions/setup-julia#119]
```
(node:1740) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
```

@SaschaMann Is this something that we need to fix?

**Comment by giordano** [comment]
Bump, I was going to report this again.

**Comment by sunoru** [comment]
It should have a simple fix so I opened a pr #133 

But it seems that our node version is too old where on some platforms `rmSync` does not exist... Should we also upgrade that as well?
